Assets/SEE/Utils/TextualDiff.cs

```diff
@@ -40,6 +41,29 @@ public static class TextualDiff
             string newRegion = FileIO.Read(newPath, newStartLine, newEndLine);
             return Diff2RichText(diff.diff_main(oldRegion, newRegion));
         }
+        /// <summary>
+        /// Returns the difference of two text regions in two version controled files.
```

Comment:
Typo. Must be `controlled`.

Reply:
The comment is confusing. There are now two text regions from two version-controlled files. There are only two input parameters of type string.

Reply:
I hope its more clear now.

Reply:
I updated the documentation without reference to any VCS.